templates/Managerpage.html

```diff
@@ -54,6 +54,7 @@
 </tr>
 </table>
 <p><input type="submit" name="Update" value="Update">
+<a href="/transactions"><button type="button">Transactions</button></a><br><br>

```

Comment:
How does this work? does it look ok? Would it look better with the `<a>` tag to be inside the `<button>` tag? Or would it look the same?